docker/README.md

```diff
@@ -62,6 +62,7 @@ The following environment variables are supported in `generate` mode:
   such as the datatase and media store. Defaults to `/data`.
 * `UID`, `GID`: the user id and group id to use for creating the data
   directories. Defaults to `991`, `991`.
+* `TZ`: the [timezone](https://en.wikipedia.org/wiki/List_of_tz_database_time_zones) the container will run with. Defaults to `UTC`.
```

Comment:
is this a useful thing in `generate` mode?

Reply:
Probably not considering it's meant to be a once off thing.

I only kept it in there because it's one of the variables that will have an impact on-going, happy to remove if that's the preference.

Reply:
I think removing it would probably make sense.

Reply:
(have gone ahead and done so)